### DISCUSSION TITLE
How to change the default landing page (homepage) to another page

### DISCUSSION THREAD

**Discussion by mvdbent** [adityatelange/hugo-PaperMod#582] [Q&A]
Can we change the default landing page (homepage) to another page?

The question is: can I do this, if so where can I configure this?

thnx.

**Comment by pixelfabs** [comment]
1. add `index.md` to `/content`
2. add
```
---
title: homepage
---

## Why do we use it?
It is a long established fact that a re
```

3. have fun!
